crates/solver/src/solver/single_order_solver.rs

```diff
@@ -50,6 +52,30 @@ pub struct Arguments {
     /// prioritization.
     #[clap(long, env, default_value = "10.0")]
     pub price_priority_max_weight: f64,
+
+    /// Configures the back off strategy for single order solvers. See
+    /// --price-estimation-rate-limiter for more details.
```

Comment:
It's a bit weird to refer to an argument description of a different binary.

Reply:
Oh... yeah you are right, didn't even notice. Will just copy-pasta it over.

Reply:
Expanded the command line documentation in d910b56cfbf02c8611d1b7990cfd760c4962ab6a